Formula/g/glow.rb

```diff
@@ -21,6 +22,8 @@ class Glow < Formula
 
   def install
     system "go", "build", *std_go_args(ldflags: "-s -w -X main.Version=#{version}")
+
+    generate_completions_from_executable(bin/"glow", "completion", base_name: "glow")
```

Comment:
```suggestion
    generate_completions_from_executable(bin/"glow", "completion")
```

Reply:
Most likely don't need to include `base_name` since the binary and formula names are the same.